ci-operator/config/quay/quay-tests/quay-quay-tests-master__ocp414-quay39.yaml

```diff
@@ -46,6 +47,8 @@ tests:
         ]
       FIREWATCH_DEFAULT_JIRA_PROJECT: LPTOCPCI
       FIREWATCH_JIRA_SERVER: https://issues.redhat.com
+      QBO_CHANNEL: stable-3.9
```

Comment:
@ascerra Is updating `QBO_CHANNEL` and `CSO_CHANNEL` versions is relevant, or the `QUAY_OPERATOR_CHANNEL` is enough? 
I specified them in the file just in case